src/Http/Http.Extensions/gen/RequestDelegateGeneratorSources.cs

```diff
@@ -102,6 +102,13 @@ private static Task ExecuteObjectResult(object? obj, HttpContext httpContext)
             }
         }
 
+        private static Func<HttpContext, StringValues> ResolveFromRouteOrQuery(string parameterName, IEnumerable<string>? routeParameterNames)
+        {
+            return routeParameterNames?.Contains(parameterName, StringComparer.OrdinalIgnoreCase) == true
+                ? (httpContext) => new StringValues(httpContext.Request.RouteValues[$"{parameterName}"]?.ToString())
+                : (httpContext) => httpContext.Request.Query[$"{parameterName}"];
```

Comment:
Is `$"{parameterName}"` necessary? Could it be just `parameterName`?

Reply:
Yep, it's not necessary.